xline/src/storage/revision.rs

```diff
@@ -11,6 +11,9 @@ pub(crate) struct KeyRevision {
     pub(crate) mod_revision: i64,
     /// Sub revision in one transaction
     pub(crate) sub_revision: i64,
+    /// Available flag
+    // true means this revision is written to storage
```

Comment:
IMO, using the present perfect tense here would be more straightforward, like "/// true means this revision has been already written to storage".

Reply:
Fixed